dashboard/lib/libraries_list.rb

```diff
@@ -0,0 +1,17 @@
+include ProjectsList
+module LibrariesList
+  class << self
+    def fetch_all_section_projects(user)
+      sections = user.sections + user.sections_as_student
+      projects = []
+      sections.each do |section|
+        projects += ProjectsList.fetch_section_projects(section).select {|project| !project[:libraryName].nil}
+      end
```

Comment:
I think the ! nil is redundant here . You could just do `project[:libraryName]`. If that doesn't work, you need a ? `!project[:libraryName].nil?`.